src/client.js

```diff
@@ -0,0 +1,107 @@
+const ContactManager = require('./contacts');
+const SenderNameManager = require('./sendernames');
+const GroupManager = require('./groups');
+const AccountManager = require('./accounts');
+const MessageManager = require('./messages');
+
+/**
+ * Represents a Nimba SMS client for interacting with Nimba SMS API resources.
+ */
+class NimbaSMSClient {
+    /**
+     * Create a new instance of the Nimba SMS client.
+     * 
+     * @constructor
+     * @param {object} options - Options for configuring the Nimba SMS client.
+     * @param {string} options.SERVICE_ID - Unique identifier for your application.
+     * @param {string} options.SECRET_TOKEN - The secret token used to authenticate your application.
+     * @memberof NimbaSMSClient
+     */
+    constructor({SERVICE_ID, SECRET_TOKEN}) {
+        /**
+         * The service ID associated with the Nimba SMS client.
+         * @type {string}
+         * @private
+         */
+        this._serviceId = SERVICE_ID;
+
+        /**
+         * The secret token associated with the Nimba SMS client.
+         * @type {string}
+         * @private
+         */
+        this._secretToken = SECRET_TOKEN;
+    }
+
+    /**
+     * Gets the manager for interacting with messages.
+     * @returns {MessageManager} The manager for messages.
+     */
+    get messages() {
+        if(!this._messagesManager) {
```

Comment:
Tu dois declarer cette variable dans le constructeur : `this._messagesManager = null` 
Ainsi que les autres sinon le code ne passera pas